js/threadRenderer.js

```diff
@@ -4,6 +4,7 @@ import { onSendButtonClick } from "./ui.js";
 import { onCancelClick } from "./ui.js";
 import { onDeleteButtonClick } from "./ui.js";
 import { convertDateString } from "./utils.js";
+import { changeLightTheme, changeDarkTheme, selectTheme } from "./themeManager.js"

```

Comment:
すみませんインポート文忘れていました…！
Approve後に原則ダメな動きをさせてしまい申し訳ございませんが、こちらの受け入れお願いさせてください…！

```suggestion
import { applyTheme } from "./themeManager.js"

```